src/components/VerticalNavigation/VerticalNavigationItem.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { ListGroup, ListGroupItem } from '../ListGroup';
+import OverlayTrigger from '../OverlayTrigger';
+import Tooltip from '../Tooltip';
+import VerticalNavigation from './VerticalNavigation';
+
+/**
+ * VerticalNavigationItem - a child element for the VerticalNavigation component
+ */
+export default class VerticalNavigationItem extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  renderBadges(badges) {
+    const { showBadges } = this.props;
+    return (
+      showBadges &&
+      badges && (
+        <div className="badge-container-pf">
+          {badges.map(badge => (
+            <OverlayTrigger
+              key={badge.badgeClass}
+              placement="right"
+              overlay={<Tooltip>{badge.tooltip}</Tooltip>}
+            >
+              <div className={cx('badge', badge.badgeClass)}>
+                {badge.count &&
+                  badge.iconStyleClass && (
+                    <span className={badge.iconStyleClass} />
+                  )}
+                {badge.count && <span>{badge.count}</span>}
+              </div>
+            </OverlayTrigger>
+          ))}
+        </div>
+      )
+    );
+  }
+
+  render() {
+    const {
+      children,
+      trackActiveState,
+      trackHoverState,
+      mobileItem,
+      iconStyleClass,
+      title,
+      badges,
+      showMobileSecondary,
+      showMobileTertiary,
+      navCollapsed,
+      handlePrimaryHover,
+      handleSecondaryHover,
+      handleTertiaryHover,
+      handlePrimaryBlur,
+      handleSecondaryBlur,
+      handleTertiaryBlur,
+      handlePrimaryClick,
+      handleSecondaryClick,
+      handleTertiaryClick,
```

Comment:
Why would an item have 3 levels of click handlers? Any one item would only ever be at one level right? Shouldn't it just have a handleClick, handleHover, handleBlur?

Not really sure why any item needs the handleHover or handleBlur callbacks. Shouldn't the hover state be maintained here? Also, you will find the need to have delays for the hover/blur handling when showing/hiding submenus.